### PR TITLE
[12.0][FIX] sale_invoice_plan, fix case advance

### PR DESCRIPTION
**Pull request by kittiu**
Invoice plan with advance, button "Create Inovice Plan" should be showing.

### CODE REVIEW

**Comment by kittiu** [comment]
@AaronHForgeFlow same fix as in v13, to show button "Create Invoice Plan" in case of advance.

**Review by AaronHForgeFlow — COMMENTED** [review]
Code review :+1: 

Cannot test on runbot because some third module is not allowing me to convert a quotation to sales order

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functionally tested locally. Thanks.

**Comment by kittiu** [comment]
@Saran440 would you help review?

**Comment by kittiu** [comment]
Travis failed because of this,
https://github.com/OCA/sale-workflow/pull/1131

Can anyone help fast forward that? then we can rebase this. :)

**Comment by kittiu** [comment]
Rebases and green travis.
@AaronHForgeFlow @JordiBForgeFlow

**Comment by AaronHForgeFlow** [comment]
:+1:  Ok for me to move forward. @kittiu  I am not PSC of this repo, but i'll try to bring reviewers here.

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Review by marcelsavegnago — APPROVED** [review]

File: sale_invoice_plan/readme/HISTORY.rst [hunk 1]
```diff
@@ -1,4 +0,0 @@
-12.0.1.0.0 (2019-03-08)
```
Comment (Saran440) [inline sale_invoice_plan/readme/HISTORY.rst:1]: why you are delete this file?

**Review by Saran440 — APPROVED** [review]
Code Review. :+1:

**Comment by kittiu** [comment]
@rousseldenis @Saran440 I added the history, I version it as "minor".
Should be ready.

**Comment by kittiu** [comment]
@rousseldenis this too, can merged?

**Comment by kittiu** [comment]
> @rousseldenis this too, can merged?

Do you still need to review this? I saw the tag. Or can this be merged?

**Pull request merged**